src/profile-logic/sanitize.js

```diff
@@ -309,12 +308,6 @@ function sanitizeThreadPII(
           );
         }
 
-        // Remove the all OS paths from FileIO markers if user wants to remove them.
-        if (currentMarker.type === 'FileIO') {
-          // Remove the filename path from marker payload.
-          markerTable.data[i] = sanitizeFileIOMarkerFilenamePath(currentMarker);
```

Comment:
We should be able to delete `sanitizeFileIOMarkerFilenamePath` function now too.

Reply:
oh that's right, thanks for recommending it!